src/domain/event/utils.ts

```diff
@@ -55,3 +56,24 @@ export const getEventStartTimeStr = (
  */
 export const isEventFree = (event: EventFieldsFragment): boolean =>
   Boolean(event.offers.find((item) => item.isFree)?.isFree);
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const getEventFields = (
+  event: EventFieldsFragment | undefined | null,
```

Comment:
This helper function is very welcome

Reply:
Yeah, we can add more fields as needed, and won't be performance event though it returns many fields that are not always needed